deploy/chart/templates/_packageserver.deployment-spec.yaml

```diff
@@ -60,5 +60,13 @@ spec:
         {{- if .Values.package.resources }}
         resources:
 {{ toYaml .Values.package.resources | indent 10 }}
-        {{- end}}
+        {{- end -}}
+        securityContext:
```

Comment:
Could you make this block conditional on the presence of `securityContext` in values.yaml?

Reply:
I can do that, but do you also want me to remove the default `runAsUser` setting in `values.yaml`?  If I don't, `securityContext` will always be set, and the conditional will be a bit meaningless.